templates/.github/workflows/publish-docs.yaml

```diff
@@ -0,0 +1,35 @@
+# This file is synced with beam-community/common-config. Any changes will be overwritten.
+
+name: Publish Docs
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: hex-publish-docs
+  cancel-in-progress: true
+
+jobs:
```

Comment:
If it's not a mix package this will be empty and result in `null` which I think GHA will error out on. You'll want this to template out to `jobs: {}` if it's not a mix package.

Reply:
I remember I did that initially, but I forgot why I moved to the inside 😭